2D3D_UnityProject/Assets/Scripts/Puzzles/Zodiac/ZodiacDisk.cs

```diff
@@ -4,6 +4,11 @@
 
 public class ZodiacDisk : ZodiacPuzzlePiece
 {
+    /// <summary>
+    /// Need to offset the x rotation of all children of the zodiac puzzle by this to account for it's default rotation being different than Unity default rotation
+    /// </summary>
+    private const float zodiacOffsetX = 90f;
```

Comment:
yes! thank you for using const magic numbers! i should do this more often myself

also good comments